storage_service/storage_service/urls.py

```diff
@@ -15,7 +14,7 @@
 urlpatterns = [
     path("", TemplateView.as_view(template_name="index.html")),
     # Uncomment the next line to enable the admin:
```

Comment:
Do you think we should remove this comment? The admin is enabled by default :shrug: